Sources/View/ViewModelView.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ViewModelView.swift
+//  Alicerce
+//
+//  Created by André Pacheco Neves on 05/05/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public protocol ViewModelView: View {
+    associatedtype ViewModel
+
```

Comment:
I left it out on purpose to avoid having to expose it publicly as a property (even though it could be `private(set)`). However we can add that and trust the users will behave and not break encapsulation 😆